portfolio/src/main/webapp/script.js

```diff
@@ -12,25 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-
-let index = 0;
+/*
+    The function showNext() allows the button to be clicked twice in order to
+    display a personal summary and a photograph. Because the second click is dependent
```

Comment:
Check the style for function comments: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments. This first sentence might be something like "Displays a personal summary and photograph one at a time when called successively."